app/e2e/tests/wessels-test-setup.ts

```diff
@@ -0,0 +1,27 @@
+import { chromium } from '@playwright/test';
+
+async function globalSetup() {
+    const browser = await chromium.launch({
+        headless: false
+    });
+    const context = await browser.newContext();
+    // Open new page
+    const page = await context.newPage();
+    await page.goto('http://localhost:8080/login');
+
+    // Login
+    await page.locator('input[type="email"]').click();
+    await page.locator('input[type="email"]').fill('mwess005@ucr.edu');
```

Comment:
I'd recommend using a fake email address like 123@somewhere.com?  That way this file can be renamed to just test-setup. Unless, is there something I'm missing?

Reply:
There is already a fake account that the DB is seeded with when running the tests. You can see it under `app/server/prisma/seed.ts`. Email: test@prytaneum.io Pass: password. Please use that account for this setup.